packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -631,7 +670,6 @@ class _DropdownMenuState<T> extends State<DropdownMenu<T>> {
       if (!_menuHasEnabledItem || !_controller.isOpen) {
         return;
       }
-      _enableFilter = false;
```

Comment:
I think removing these `_enableFilter = false` will change the original logic. Originally, when we open the menu, and do "up" and "down" key traverse, the text input will be changed based on the traversed text, but we still want to show the complete menu items instead of filtering all of them. So we probably want to have this logic for now as this should not be related to the new property.

Reply:
I decided to remove it just because the variable `_enableFilter` was never used anywhere beside setting its value, which confused me. Anyway, @QuncCccccc  if you think I should put it back, let me know and I will revert the changes.

Reply:
@QuncCccccc ping

Reply:
done